spring-cloud-sleuth-instrumentation/src/main/java/org/springframework/cloud/sleuth/instrument/web/TraceWebFilter.java

```diff
@@ -69,11 +71,14 @@
 
 	private final HttpServerHandler handler;
 
+	private final CurrentTraceContext currentTraceContext;
+
 	private int order;
 
-	public TraceWebFilter(Tracer tracer, HttpServerHandler handler) {
+	public TraceWebFilter(Tracer tracer, HttpServerHandler handler, CurrentTraceContext currentTraceContext) {
```

Comment:
This is a breaking change. We need to create another constructor instead of modifying this one. We can mark the current one as deprecated and ensure that we're using the new one everywhere